test/e2e/allocation.new.test.js

```diff
@@ -0,0 +1,52 @@
+import { Selector, t } from 'testcafe'
+
+import { pmuUser } from './_roles'
+import { allocations, newAllocation } from './_routes'
+import { allocationDetails, allocationCriteria, page } from './pages/'
+
+const selectors = {
+  cancelJourney: {
+    linkToFirstAllocation: Selector('.govuk-table td:first-of-type a'),
+    cancelLink: Selector('.app-link--destructive').withText(
+      'Cancel allocation'
+    ),
+    statusHeading: Selector(
+      '.app-message.app-message--temporary .app-message__heading'
+    ).withText('Allocation cancelled'),
+  },
+}
+
+async function createAllocation() {
+  await allocationDetails.fill()
+  await page.submitForm()
+  await allocationCriteria.fill()
+  await page.submitForm()
+  return t
+    .expect(page.getCurrentUrl())
+    .match(/\/allocation\/[\w]{8}(-[\w]{4}){3}-[\w]{12}\/confirmation$/)
+}
+
+fixture('New PMU allocation').beforeEach(async t => {
+  await t.useRole(pmuUser).navigateTo(newAllocation)
+})
+test('by default', t => {
```

Comment:
This test doesn't really feel very clear and descriptive. In the test output on CI it would just read:

```
New PMU allocation
  ✓ by default
```

Maybe we can follow the pattern that has been used elsewhere:

```
New allocation
  ✓ with all allocation criteria
```

It might also be good to test variations of the criteria screen within this section.